implementations/rust/ockam/ockam_node/src/storage/database/migrations/node_migrations/sql/sqlite/20250409100000_make_resource_type_optional.sql

```diff
@@ -0,0 +1,15 @@
+ALTER TABLE resource RENAME TO resource_old;
+DROP INDEX IF EXISTS resource_index;
+
+-- Create a new table for resource to resource type mapping
+CREATE TABLE resource
+(
+    resource_name   TEXT NOT NULL, -- resource name
+    resource_type   TEXT,          -- resource type
+    node_name       TEXT NOT NULL  -- node name
+);
+CREATE UNIQUE INDEX resource_index ON resource (node_name, resource_name);
+
+INSERT INTO resource (resource_name, resource_type, node_name)
+SELECT resource_name, resource_type, node_name FROM resource_old;
+
```

Comment:
I think that you need to drop `resource_old` at the end.

Reply:
Indeed, thank you !